logs-bucket/main.tf

```diff
@@ -0,0 +1,55 @@
+provider "aws" {
+  region = var.aws_region
+}
+
+locals {
+  env_by_workspace = {
+    "testnet-2.0" = "testnet-20"
+    "mainnet-2.0" = "mainnet-20"
```

Comment:
In other places we have just `mainnet` and `testnet` 🤔 . I am wondering if there is a need to specify the version if 1.0 is killed.